style.css

```diff
@@ -43,24 +43,205 @@ nav {
   position: absolute;
   right: 1rem;
   display: none;
+  top: 1.5rem;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
+}
+
+.close {
+  display: none;
 }
 
 @media (max-width: 500px) {
   .menu-button {
     display: flex;
   }
 
+  .close-button {
+    width: 20px;
+    height: 20px;
+    left: 5px;
+    top: 5px;
+  }
+
+  .close {
+    display: flex;
+    height: 50px;
+    width: 100%;
+    border-radius: 8px;
+    justify-content: flex-end;
+    padding: 25px;
+  }
+
   .navbar-links {
     display: none;
+    width: 105%;
+    background: #6070ff;
+    height: 100vh;
+    margin-left: -20px;
   }
 
   .navbar {
+    flex-direction: column;
+    align-items: flex-start;
```

Comment:
- [ ] Please from the project requirements, you should stick to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=YHGkZAlalTNDuhZx-0) as much as possible. But as you can see you have some differences in the design. 

You can see the differences below

- The background of the menu popup should be blurred
- You should have a close button on the menu popup
- You should have your menu links aligned to the left and on the same line
- You should hot have vertical scroll on the menu popup

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2022-11-23 10-14-42](https://user-images.githubusercontent.com/92303891/203524158-575b93d1-2bad-4c3c-9ce2-9c027447b32d.png) | ![Screenshot from 2022-11-23 10-16-15](https://user-images.githubusercontent.com/92303891/203524200-cf459402-77cd-493b-b5f4-24d9bb313fab.png) |